edge-middleware/feature-flag-hypertune/README.md

```diff
@@ -0,0 +1,38 @@
+# Feature flags, A/B testing and app configuration with Hypertune and Vercel Edge Config
+
+[Hypertune](https://www.hypertune.com/) is the most powerful feature flag, A/B testing and app configuration platform. Built for TypeScript, Next.JS, Vercel and the modern stack with full type-safety and Git version control.
```

Comment:
Seeing that this is going into a Vercel hosted example, I'd refrain from language such as "most powerful" as it might come across as Vercel making a recommendation